addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVWebSocket.java

```diff
@@ -0,0 +1,386 @@
+                updateChannel(ValloxMVBindingConstants.CHANNEL_FAN_SPEED_SUPPLY, new DecimalType(bdFanspeedSupply));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_TEMPERATURE_INSIDE,
+                        new QuantityType<>(bdTempInside, SIUnits.CELSIUS));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_TEMPERATURE_OUTSIDE,
+                        new QuantityType<>(bdTempOutside, SIUnits.CELSIUS));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_TEMPERATURE_EXHAUST,
+                        new QuantityType<>(bdTempExhaust, SIUnits.CELSIUS));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_TEMPERATURE_INCOMING_BEFORE_HEATING,
+                        new QuantityType<>(bdTempIncomingBeforeHeating, SIUnits.CELSIUS));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_TEMPERATURE_INCOMING,
+                        new QuantityType<>(bdTempIncoming, SIUnits.CELSIUS));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_HUMIDITY,
+                        new QuantityType<>(bdHumidity, SmartHomeUnits.PERCENT));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_CELLSTATE, new DecimalType(bdCellstate));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_UPTIME_YEARS, new DecimalType(bdUptimeYears));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_UPTIME_HOURS, new DecimalType(bdUptimeHours));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_UPTIME_HOURS_CURRENT,
+                        new DecimalType(bdUptimeHoursCurrent));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_EXTR_FAN_BALANCE_BASE,
+                        new QuantityType<>(bdExtrFanBalanceBase, SmartHomeUnits.PERCENT));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_SUPP_FAN_BALANCE_BASE,
+                        new QuantityType<>(bdSuppFanBalanceBase, SmartHomeUnits.PERCENT));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_HOME_SPEED_SETTING,
+                        new QuantityType<>(bdHomeSpeedSetting, SmartHomeUnits.PERCENT));
+                updateChannel(ValloxMVBindingConstants.CHANNEL_HOME_AIR_TEMP_TARGET,
+                        new QuantityType<>(bdHomeAirTempTarget, SIUnits.CELSIUS));
+
+                voHandler.updateStatus(ThingStatus.ONLINE);
+                voHandler.dataUpdated();
+            } catch (IOException e) {
+                connectionError(e);
+            } catch (Exception e) {
+                logger.error("Error: {}", e.getMessage());
+            }
+            logger.debug("Print final");
+        }
+
+        private void updateChannel(String strChannelName, State state) {
+            Channel channel = voHandler.getThing().getChannel(strChannelName);
+            if (channel != null) {
+                voHandler.updateState(channel.getUID(), state);
```

Comment:
AFAIR some bindings use some static value here, maybe pointing immediately to the UID, it seems way more optimal than mapping strings to channels to uid's every time.